tests/README.md

```diff
@@ -142,3 +142,23 @@ To run specific tests inside a suite:
 ```console
 _output/tests/linux_amd64/integration -test.v -test.timeout 1800s -test.run CephSmokeSuite -testify.m TestARookClusterInstallation_SmokeTest
 ```
+
+### To run tests on OpenShift environment
+
+- You have to setup OpenShift environment by your own and export KUBECONFIG before executing the tests.
+- Make sure `oc` executable file is in your PATH.
+- Currenttly we are enabling only `CephSmokeSuite`  on OpenShift environment.
+- To start the test we need to set few environment variables :-
+```
+export TEST_PLATFORM=openshift
+export TEST_ENV_NAME=openshift
+export storageClassName=gp2
+export usePVC=true
+export TEST_BASE_DIR=/var/lib/rook
```

Comment:
Do we have to set this? Or can we just leave it blank and use the default?

Reply:
Which one specific?

Reply:
`TEST_BASE_DIR`, I wonder if we can just use the default set inside the tests. If the default doesn't work, maybe it should be a path under `/tmp`

Reply:
`\tmp` works. Done.